sdk/compatibility/versions/UpdateVersions.hs

```diff
@@ -78,9 +79,7 @@ renderVersionsFile (Versions (Set.toAscList -> versions)) checksums =
         , "        \"daml_types\": " <> renderDigest damlTypesHash <> ","
         , "        \"daml_ledger\": " <> renderDigest damlLedgerHash <> ","
         , "        \"daml_react\": " <> renderDigest damlReactHash <> ","
-        ]
-      , [ "        \"create_daml_app_patch\": " <> renderDigest hash <> ","
-        | Just hash <- [mbCreateDamlAppPatchHash]
+        , "        \"create_daml_app_patch\": " <> renderDigest createDamlAppPatchHash <> ","
```

Comment:
We are no longer testing any version that does not have that patch, so it does not need to be a Maybe anymore. This is the first place impacted by that change, though it obviously ripples through a few other places because of type coupling.